contracts/proxy.scilla

```diff
@@ -120,22 +120,23 @@ transition update_contractmaxstake (max_stake : Uint128)
     send msgs
 end
 
-(* @dev: Adds new ssn to ssnlist. Used by verifier only. *)
-(* @param ssnaddr: Address of the ssn to be added *)
-(* @param urlraw: string representing "ip:port" of the ssn serving raw api request *)
-(* @param urlapi: string representing url exposed by ssn serving public api request *)
-(* @param buffered_deposit: Any buffered stake deposit *)
-
 (* @dev: Adds new ssn to ssnlist. Used by admin only. *)
 (* @param ssnaddr: Address of the ssn to be added *)
 (* @param stake_amount: Stake amount of ssn *)
 (* @param rewards: Rewards of the ssn to be added *)
 (* @param urlraw: string representing "ip:port" of the ssn serving raw api request *)
 (* @param urlapi: string representing url exposed by ssn serving public api request *)
 (* @param buffered_deposit: Any buffered stake deposit *)
-transition add_ssn (ssnaddr : ByStr20, stake_amount : Uint128, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128)
+transition add_ssn (ssnaddr : ByStr20, rewards : Uint128, urlraw : String, urlapi : String, buffered_deposit : Uint128)
```

Comment:
bufffered_deposit should be zero right?

Reply:
reward should be 0 too

Reply:
Yes, I will remove `buffered_deposit ` too. Thanks.